omp/test/preconditioner/jacobi_kernels.cpp

```diff
@@ -562,6 +588,35 @@ TEST_F(Jacobi, OmpPreconditionerEquivalentToRefWithAdaptivePrecision)
 }
 
 
+TEST_F(Jacobi, OmpTransposedPreconditionerEquivalentToRefWithAdaptivePrecision)
+{
+    initialize_data({0, 11, 24, 33, 45, 55, 67, 70, 80, 92, 100},
+                    {sp, sp, dp, dp, tp, tp, qp, qp, hp, dp, up}, {}, 13, 97,
+                    99);
+
+    auto bj = bj_factory->generate(mtx);
+    auto d_bj = d_bj_factory->generate(mtx);
+
+    GKO_ASSERT_MTX_NEAR(gko::as<Bj>(d_bj->transpose()),
+                        gko::as<Bj>(bj->transpose()), 1e-1);
```

Comment:
In this test, 1e-1 comes from the `generate` not the transpose.
maybe both use reference generated result and do transpose on device/ref. The result should be the same.

Reply:
So you mean d_bj->copy_from(bj) instead of generating it?

Reply:
Yes